go/vt/sqlparser/sql.y

```diff
@@ -950,10 +985,157 @@ value_expression:
   {
     $$ = $1
   }
+
 | VALUES openb sql_id closeb
   {
     $$ = &ValuesFuncExpr{Name: $3}
   }
+| CONVERT openb expression ',' convert_type closeb
+  {
+    $$ = &ConvertExpr{Expr: $3, Type: $5}
+  }
+| CAST openb expression AS convert_type closeb
+  {
+    $$ = &ConvertExpr{Expr: $3, Type: $5}
+  }
+| MATCH openb column_list closeb AGAINST openb value_expression match_option closeb
+{
+  $$ = &MatchExpr{Columns: $3, Expr: $7, Option: $8}
+}
+
+match_option:
```

Comment:
opt means optional (and should be prefix, but we're doing suffix)

though this option is, in fact, optional, i thought opt_match_option was a bit too much